### ISSUE TITLE
Achievements: Bug in Features Pictures number

### ISSUE THREAD

**Issue by edwa2** [commons-app/apps-android-commons#3788]
**Summary:** 

The Featured Pictures number in achievements doesn't accurately reflect the number of featured pictures uploaded by a user

**Steps to reproduce:** 

Image attached

**Device and Android version:** 

Galaxy S9+, Android 10
 
**Commons app version:** 

API level: 29
Android version: 10
Device manufacturer: samsung
Device model: SM-G965U
Device: star2qltesq
Network type: wifi
App version name: 2.13~d950f7219
User name: EoRdE6
[https://commons.wikimedia.org/wiki/User:EoRdE6](https://commons.wikimedia.org/wiki/User:EoRdE6)

**Screen-shots:** 

![Screenshot_20200528-132859_Commons](https://user-images.githubusercontent.com/66191344/83343457-734ff400-a2c8-11ea-95d5-ea3d7862d6fe.png)

**Would you like to work on the issue?**

n/a

**Comment by rhulkb27** [comment]
I would like to work on this issue, I will try to repro it tonight. Is there a staging environment I can use? I currently do not have any featured photos on my account. @nicolas-raoul @edwa2 .

**Comment by nicolas-raoul** [comment]
@rhulkb27 Thanks for working on that!

Actually featured images are super rare haha
There is a staging environment but I don't think it has featured images.

My advice:

1. Using logcat see what network request is sent to get this number
2. Run the same request from curl or your browser
3. Replace the username with names of users who have and don't have featured images
4. Post your findings here.

Thanks! :-)

**Comment by gerardolgvr** [comment]
Hello @nicolas-raoul , can I help in this issue?  I have followed the steps you describe below and I find the following information. Just I'm not sure what's the correct field should be rendered. 

I find these urls in the logcat: 
- https://tools.wmflabs.org/urbanecmbot/commonsmisc/wikidataedits.py?user=EoRdE6
- https://tools.wmflabs.org/urbanecmbot/commonsmisc/uploadsbyuser.py?user=EoRdE6
- https://tools.wmflabs.org/urbanecmbot/commonsmisc/feedback.py?user=EoRdE6 

Now, I have test the url with 2 users and I got these results: 

![image](https://user-images.githubusercontent.com/20119298/84807534-011c2680-afcd-11ea-880e-cfab18947aa8.png)

![image](https://user-images.githubusercontent.com/20119298/84807575-11cc9c80-afcd-11ea-9c01-23c75c15e5ca.png)


Can you give me a hint what is the correct field and if it's ok the results for these users o if I have to test with other users (Can you share how i can find more usernames?) 

Thanks

**Comment by nicolas-raoul** [comment]
Featured_pictures_on_Wikimedia_Commons sounds like the correct number to use :-)

You can find many usernames here: https://commons.wikimedia.org/wiki/Special:RecentChanges?hidebots=1&translations=filter&hidecategorization=1&hideWikibase=1&limit=500&days=7&urlversion=2 99.99% of them do not have any featured picture I guess.

To find usernames of people who have featured pictures, the easiest is to read usernames of the authors of the pictures at https://commons.wikimedia.org/wiki/Commons:Featured_pictures for instance "Sven Damerow" for https://commons.wikimedia.org/w/index.php?title=File:Gro%C3%9Fer_Blaupfeil_-_Weibchen.jpg&action=history

**Comment by gerardolgvr** [comment]
Thanks @nicolas-raoul  for the information!

I'm gonna review the information, any update I'm gonna post it here

**Comment by gerardolgvr** [comment]
![image](https://user-images.githubusercontent.com/20119298/84845982-70b50480-b013-11ea-94ce-393fffe90896.png)

Hello @nicolas-raoul again heheh

I found where the problem is located, at the image is the response for the url:
https://tools.wmflabs.org/urbanecmbot/commonsmisc/feedback.py?user=EoRdE6

As you can see the field "featuredImages" is a sum from :
"Quality_images" and "Featured_pictures_on_Wikimedia_Commons"

So, the solution is only return the following field :  "Featured_pictures_on_Wikimedia_Commons"

Seems well to you?

**Comment by nicolas-raoul** [comment]
Actually, I would be in favor of just renaming the label to "Quality Images" instead of "Featured Images".

0.394% of Commons images are Quality Images.
0.022% of Commons images are Featured Images.

**Comment by gerardolgvr** [comment]
Hello @nicolas-raoul again

![image](https://user-images.githubusercontent.com/20119298/84850558-a7901800-b01d-11ea-9a68-890bbf400779.png)

I review the code a explore the code and found that the problem is ar Achievements.kt. As you can notice the field "featuredImages" is the result of sum: "qualityImages" and "featuredPicturesOnWikimediaCommons" . So, the solution is only use "featuredPicturesOnWikimediaCommons" in the constructor. 

Seems well to you? I checked the code and this response is only used at AchievementsActivity so we are not affecting other code with this change. If you have a comment or I'm not considering something feel free to comment

**Comment by nicolas-raoul** [comment]
Actually, if you have time could you split into two different statistics in the UI?
It makes sense to show both the number of features images, and also the number of quality images. :-)

**Comment by gerardolgvr** [comment]
Sure, sounds a good idea.

I can do the changes, I can add it just below the first one 

About the icon:
- I can search for a new one as a proposal if you want or use a particular icon you prefer (I saw some icons in drawable folder). I'm gonna do the changes to share you a SS

**Comment by gerardolgvr** [comment]
I added the changes requested: 
![WhatsApp Image 2020-06-17 at 3 37 21 PM](https://user-images.githubusercontent.com/20119298/84948523-77925480-b0b1-11ea-9600-348473b71b7d.jpeg)


I found an icon but I'm not sure about it. Here are some icons that I found: 
![image](https://user-images.githubusercontent.com/20119298/84948864-ecfe2500-b0b1-11ea-88ae-72f789164f54.png).  

What do you think?

**Comment by nicolas-raoul** [comment]
Wonderful!
`5` and `10` are just debug values, right?

The icon you chose is nice I think :-) Where did you find it? We must use only public domain icons.

**Comment by rhulkb27** [comment]
hi @nicolas-raoul It looks like @gerardolgvr has made some good progress on this, more then I had made so far (admittedly not that much). As such, please feel free to re assign the issue to him; I will look for another issue to work on.

Thanks!

**Comment by nicolas-raoul** [comment]
@rhulkb27 Understood, thanks! :-)

@gerardolgvr Actually would you mind using this icon? https://upload.wikimedia.org/wikipedia/commons/8/8c/Quality_images_logo.svg
It is the official icon for quality images, as seen at https://commons.wikimedia.org/wiki/Commons:Quality_images Thanks! :-)

**Comment by gerardolgvr** [comment]
@rhulkb27 thanks you, no problem :) 

Yup, just to debug the code. 
I found those icons at flaticon.com 

Sure, I can add that icon @nicolas-raoul . I have to say that I added a brief desciption from https://commons.wikimedia.org/wiki/Commons:Quality_images to lauch the proper alert and probably we must add the proper link, associated according this #3433 but I think that the pr associate to #3433 must be merged first in order to link to the proper help link. We can checked later :)

**Comment by gerardolgvr** [comment]
I think this is ready for a PR, if there isn't anything to add. How it seems to you ? @nicolas-raoul 

![WhatsApp Image 2020-06-18 at 12 38 22 AM](https://user-images.githubusercontent.com/20119298/84982554-9b2ebc80-b0fc-11ea-9d07-ac703851a0ee.jpeg)

**Comment by nicolas-raoul** [comment]
Is there an easy way to make the logo not be cropped, for instance by resizing it?
If not, that can be the topic of a future issue so don't worry too much, and submit your PR when you want :-)

**Comment by gerardolgvr** [comment]
Sure, I have fixed the image problem. The problem was when I was importing the image. Android studio face problems when image is converted to xml, especially when they have kind of gradients or some stuffs like that: 
https://stackoverflow.com/questions/59610243/android-studio-does-not-import-all-parts-of-an-svg-vector

So I imported the logo as png

Here's the final result.

![WhatsApp Image 2020-06-18 at 9 13 22 AM](https://user-images.githubusercontent.com/20119298/85031533-68101b80-b144-11ea-8ead-37faedb17e95.jpeg)
